slides/Symmetric Cryptography.tex

```diff
@@ -0,0 +1,507 @@
+				20 &U&V&W&X&Y&Z&A&B&C&D&E&F&G&H&I&J&K&L&M&N&O&P&Q&R&S&T \\
+				21 &V&W&X&Y&Z&A&B&C&D&E&F&G&H&I&J&K&L&M&N&O&P&Q&R&S&T&U \\
+				22 &W&X&Y&Z&A&B&C&D&E&F&G&H&I&J&K&L&M&N&O&P&Q&R&S&T&U&V \\
+				23 &X&Y&Z&A&B&C&D&E&F&G&H&I&J&K&L&M&N&O&P&Q&R&S&T&U&V&W \\
+				24 &Y&Z&A&B&C&D&E&F&G&H&I&J&K&L&M&N&O&P&Q&R&S&T&U&V&W&X \\
+				25 &Z&A&B&C&D&E&F&G&H&I&J&K&L&M&N&O&P&Q&R&S&T&U&V&W&X&Y\\
+				26 &A&B&C&D&E&F&G&H&I&J&K&L&M&N&O&P&Q&R&S&T&U&V&W&X&Y&Z\\
+				\hline
+			\end{tabular}
+		}
+		\caption{Encryption of \textit{b}, \textit{l} and \textit{o} with code word "\textit{CIF}"}
+	\end{table}
+	\begin{tikzpicture}[overlay]
+		% 2,b
+		\draw[focus, line width=1.5pt] (1.35,7.27) ellipse (0.15cm and 0.15cm);
+		% 5,o
+		\draw[focus, line width=1.5pt] (6.26,6.56) ellipse (0.15cm and 0.15cm);
+		% 8,l
+		\draw[focus, line width=1.5pt] (5.14,5.88) ellipse (0.15cm and 0.15cm);
+	\end{tikzpicture}
+\end{frame}
+%%%
+
+%%%
+\begin{frame}{Monoalphabetic vs. Polyalphabetic}
+	\begin{table}
+		\centering
+		\begin{tabular}{l c c c}
+			\hline 
+			Method & Plain & & Cipher\\
+			\hline
+			Monoalphabetic & 1 & $\leftrightarrow$ & 1\\
+			Homophone & 1 & $\leftrightarrow$ & N\\
+			Polyalphabetic & N & $\leftrightarrow$ & N\\
+			\hline
+		\end{tabular}
+	\end{table}
+	\vspace{0.5cm}
+	\begin{itemize}
+		\item<1-> Monoalphabetic: every cipher letter stands for exactly one plain text letter
+		\item<2-> Homophone: multiple cipher symbols can stand for one plain text letter
+		\item<3-> Polyalphabetic: the same cipher letter can stand for multiple plain text letters and a plain text letter can be encrypted by multiple cipher letters
+	\end{itemize}
+\end{frame}
+%%%
+
+%%%
+\begin{frame}{Breaking Polyalphabetic Substitution}
+	\begin{itemize}
+		\item<1-> Simple frequency analysis not possible anymore
+		\item<2-> Mid 19th century: vulnerability discovered
+		\item<3-> Use repetition of code word as starting point
+		\item<4-> Find length of code word and use frequency anaylsis on every x-th cipher letter
+	\end{itemize}
+\end{frame}
+%%%
+
+%%%
+\begin{frame}{Breaking Polyalphabetic Substitution}
+	\begin{table}
+		\centering
+		\resizebox{10.5cm}{!} {
+			\begin{tabular}{l c c c c c c c c c c c c c c c c c c c c c c c c}
+				\hline
+				Code word &K&I&N&G&K&I&N&G&K&I&N&G&K&I&N&G&K&I&N&G&K&I&N&G\\
+				Plain text &t&h&e&s&u&n&a&n&d&t&h&e&m&a&n&i&n&t&h&e&m&o&o&n \\
+				Cipher text &D&P&R&Y&E&V&N&T&N&B&U&K&W&I&A&O&X&B&U&K&W&W&B&T\\
+				\hline
+			\end{tabular}
+		}
+		\caption{Encryption with Vigenère Square. Based on \cite{singh1999}}
+	\end{table}
+	\uncover<3->{
+		\begin{tikzpicture}[overlay]
+			\draw[focus, line width=1.5pt] (2.04,2.1) ellipse (0.6cm and 0.75cm);
+			\draw[focus, line width=1.5pt] (5.42,2.1) ellipse (0.6cm and 0.75cm);
+			\draw[focus, line width=1.5pt] (8.4,2.1) ellipse (0.6cm and 0.75cm);
+		\end{tikzpicture}
+	}
+	\begin{itemize}
+		\item<2-> "the" can be enrcrypted in four ways: DPR, BUK, GNO, ZRM
```

Comment:
```suggestion
		\item<2-> using a code word with 4 letters, "the" can be encrypted in four ways: DPR, BUK, GNO, ZRM
```

Reply:
I would like to keep the four alternatives how "the" can be encrypted in this example.